rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -2511,4 +2511,21 @@ class ReduceSpec extends FlatSpec with Matchers with AppendedClues with Persiste
 
     task.runSyncUnsafe(timeout)
   }
+
+  "term split size" should "be limited to Short.MaxValue" in {
+    implicit val errorLog = new ErrorLog[Task]()
+    val sends             = Seq.fill(Short.MaxValue + 2)(Send())
```

Comment:
The minimum failing case should be `Short.MaxValue + 1`.

We should also have a positive test showing that `Short.MaxValue` pars is OK

Reply:
Do you think on test I mentioned in the description? Can we make it faster?